static/app/views/releases/detail/overview/sidebar/commitAuthorBreakdown.tsx

```diff
@@ -85,27 +86,28 @@ class CommitAuthorBreakdown extends AsyncComponent<Props, State> {
 
     return (
       <Wrapper>
```

Comment:
Should the `SidebarSection` have some predefined margin so that they are easily stackable on top of each other?
We could get rid of all these Wrapper components and ensure that the spacing is consistent everywhere.